windows/jitsi-meet/index.html

```diff
@@ -0,0 +1,15 @@
+ <!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" type="text/css" href="../../css/welcome.css">
+        <script src="https://meet.jit.si/external_api.js"></script>
+        <script>
+            require("./render.js");
+        </script> 
+    </head>
+    <body>
```

Comment:
The rest of the UI components (like the calendar) are going to be React based. Let's start using React right here right now! :-)